### ISSUE TITLE
Multiple consecutive invocations/accesses to members should be placed with same indentation when spanning multiple lines

### ISSUE THREAD

**Issue by RalfKoban** [RalfKoban/MiKo-Analyzers#529]
We should file a warning in case multiple consecutive invocations that span multiple lines do not have the same indentation.

Following example should report a violation:
```c#
abcde.DoSomething()
             .DoSomethingAdditionally()
  .AndSomethingMore();
```

It should be fixed to following:
```c#
abcde.DoSomething()
     .DoSomethingAdditionally()
     .AndSomethingMore();
```
